### PR TITLE
Fix cfg clippy for libafl_qemu

### PR DESCRIPTION
**Pull request by rmalmain**

### CODE REVIEW

File: libafl_qemu/Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
 default = ["fork", "build_libgasan", "build_libqasan", "serdeany_autoreg", "injections"]
-clippy = [] # special feature for clippy, don't use in normal projects§
+clippy = ["libafl_qemu_sys/clippy"] # special feature for clippy, don't use in normal projects§
```
Comment (domenukk) [inline libafl_qemu/Cargo.toml:20]: the dollar can probably go :D

File: libafl_qemu/Cargo.toml [hunk 2]
```diff
@@ -17,7 +17,7 @@ rustdoc-args = ["--cfg", "docsrs"]
...
 document-features = ["dep:document-features"]
 paranoid_debug = ["libafl_qemu_sys/paranoid_debug"] # Will perform as many checks as possible. The target will be greatly slowed down.
```
Comment (domenukk) [inline libafl_qemu/Cargo.toml:22]: The comment should be above with `##` then it'll show up in the docs.
Also, move it below Feature Flags (?)

Comment (rmalmain) [reply libafl_qemu/Cargo.toml:21]: Should it really show up in the doc if it's an internal feature though?

**Comment by addisoncrump** [comment]
I cancelled the CI since the clippy pass failed.

**Pull request merged**